### ISSUE TITLE
GPT2 Tokenizer 

### ISSUE THREAD

**Issue by Saisuchith** [huggingface/transformers#1435]
I want to know the pad token value for the gpt2 tokenizer. I have checked the [vocab](https://s3.amazonaws.com/models.huggingface.co/bert/gpt2-vocab.json) but couldn't find any.

Thanks,
Suchith

**Comment by LysandreJik** [comment]
Hi! GPT-2 doesn't use padding ids in its tokenizer. In order to manage padding, you should use the `attention_mask` detailed in the [documentation](https://huggingface.co/transformers/model_doc/gpt2.html#transformers.GPT2Model).
